src/caret_analyze/architecture/architecture.py

```diff
@@ -381,6 +409,19 @@ def insert_publisher_callback(self, node_name: str,
 
     def insert_variable_passing(self, node_name: str,
                                 callback_name_write: str, callback_name_read: str) -> None:
+        """
+        Insert variable passing in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node to insert variable passing
+        callback_name_write : str
+            name of write callback in variable passing to insert
+        callback_name_read : str
+            name of read callback in variable passing to insert
```

Comment:
```suggestion
        Insert variable_passing in "node_name" node.

        Parameters
        ----------
        node_name : str
            name of target node to insert variable_passing
        callback_name_write : str
            name of write callback in variable_passing to insert
        callback_name_read : str
            name of read callback in variable_passing to insert
```

Reply:
Fix it in [c5a7bd9](https://github.com/tier4/CARET_analyze/pull/300/commits/c5a7bd978be6ff8d12c907120a79795467a95842).